flow360/component/simulation/models/volume_models.py

```diff
@@ -194,65 +231,148 @@ def validate_consistent_array_length(cls, values):
+        description="The 3D arrays specifying the drag coefficient."
+    )
 
 
 # pylint: disable=no-member
 class BETDisk(Flow360BaseModel):
-    """Same as Flow360Param BETDisk.
+    """:class:`BETDisk` class for defining the Blade Element Theory (BET) model inputs.
+    For detailed information on the parameters, please refer to the :ref:`BET knowledge Base <knowledge_base_BETDisks>`.
+    To generate the sectional polars the BET translators can be used which are
+    outlined :ref:`here <BET_Translators>`
+    with best-practices for the sectional polars inputs available :ref:`here <secPolars_bestPractices>`.
+    A case study of the XV-15 rotor using the steady BET Disk method is available
+    in :ref:`Case Studies <XV15BETDisk_caseStudy>`.
+    Because a transient BET Line simulation is simply a time-accurate version of a steady-state
+    BET Disk simulation, most of the parameters below are applicable to both methods.
     Note that `center_of_rotation`, `axis_of_rotation`, `radius`, `thickness` can be acquired from `entity`
     so they are not required anymore.
     """
 
-    name: Optional[str] = pd.Field(None)
+    name: Optional[str] = pd.Field(None, description="Name of the `BETDisk` model")
     type: Literal["BETDisk"] = pd.Field("BETDisk", frozen=True)
     entities: EntityList[Cylinder] = pd.Field(alias="volumes")
 
-    rotation_direction_rule: Literal["leftHand", "rightHand"] = pd.Field("rightHand")
-    number_of_blades: pd.StrictInt = pd.Field(gt=0, le=10)
-    omega: AngularVelocityType.NonNegative = pd.Field()
-    chord_ref: LengthType.Positive = pd.Field()
-    n_loading_nodes: pd.StrictInt = pd.Field(gt=0, le=1000)
-    blade_line_chord: LengthType.NonNegative = pd.Field(0)
-    initial_blade_direction: Optional[Axis] = pd.Field(None)
-    tip_gap: Union[Literal["inf"], LengthType.NonNegative] = pd.Field("inf")
-    mach_numbers: List[pd.NonNegativeFloat] = pd.Field()
-    reynolds_numbers: List[pd.PositiveFloat] = pd.Field()
-    alphas: List[float] = pd.Field()
-    twists: List[BETDiskTwist] = pd.Field()
-    chords: List[BETDiskChord] = pd.Field()
-    sectional_polars: List[BETDiskSectionalPolar] = pd.Field()
-    sectional_radiuses: List[float] = pd.Field()
+    rotation_direction_rule: Literal["leftHand", "rightHand"] = pd.Field(
+        "rightHand",
+        description='The rule for rotation direction and thrust direction, "rightHand" or "leftHand".',
+    )
+    number_of_blades: pd.StrictInt = pd.Field(gt=0, le=10, description="Number of blades to model")
+    omega: AngularVelocityType.NonNegative = pd.Field(
+        description="Nondimensional rotating speed, radians/nondim-unit-time, "
+        + "= :math:`\\frac{\\Omega*L_{gridUnit}}{C_\\infty}`"
+    )
+    chord_ref: LengthType.Positive = pd.Field(
+        description="Nondimensional reference chord used to compute sectional blade loadings"
+    )
+    n_loading_nodes: pd.StrictInt = pd.Field(
```

Comment:
We probabaly need to change how we specify the range:
![image](https://github.com/user-attachments/assets/8700d2cb-f335-49d5-be03-1e54f6ae4eab)
VS
![image](https://github.com/user-attachments/assets/9b5cbb77-1ee6-447a-ae16-9f5090aaeec3)

Reply:
We have tested using Annotated but it does not work. We will improve this in the future.